### DISCUSSION TITLE
Natural Sorting, come applicarlo ai valori di una cella?

### DISCUSSION THREAD

**Discussion by pigreco** [ondata/guidaVisiData#51] [Domande] [answered]
In questo periodo ho avuto necessità di applicate quanto in oggetto a dataset così strutturati:

id|name|list|natural_sorted_list
--|------|---|--------
1|SARTI-GRILLETTO|16/A-0|"0,16/A"
15|VIA COSTA AL BAGNO|13-4-6-9/A-2-1|"1,2,4,6,9/A,13"
26|VIA VAL DEI CEPPI|6-1/B-4-5-1/A-2-10-1-7|"1,1/A,1/B,2,4,5,6,7,10"
33|VIA ANDREA|1-1/A-10-1r-2/10-10/1A2-10/A-1000/C1-SNC|"1,1/A,1r,2/10,10,10/1A2,10/A,1000/C1,SNC"

file di esempio:
[ns.csv](https://github.com/ondata/guidaVisiData/files/8056548/ns.csv)

**Comment by aborruso** [comment, marked as answer]
Grazie mille per l'ispirazione.

Trovi tutto in una ricetta dedicata https://ondata.github.io/guidaVisiData/ricette/#applicare-il-natural-sorting-ai-contenuti-di-una-cella

**Reply by pigreco** [reply]
Che meraviglia, sotto un video demo, grazie mille @aborruso :

https://user-images.githubusercontent.com/7631137/153773199-18bfb446-df4f-4f28-8f51-e1dc170c197a.mp4

**Reply by aborruso** [reply]
Sul tema farò un tutorial più ampio.
E citerò anche questo video

**Comment by pigreco** [comment]
Ho appena seguito la ricetta,
funziona benissimo.

grazie mille!
